cdap-distributions/src/Dockerfile

```diff
@@ -43,7 +43,9 @@ COPY packer/scripts /tmp/scripts
 COPY packer/files /tmp/files
 
 # Install Chef, setup APT, run Chef cdap::sdk recipe, then clean up
-RUN curl -vL http://chef.io/chef/install.sh | bash -s -- -v 13.8.5 && \
+RUN wget https://packages.chef.io/files/stable/chef/13.8.5/ubuntu/16.04/chef_13.8.5-1_amd64.deb && \
```

Comment:
Can we just use curl here to avoid needing to install wget?

Reply:
ack